test/unit/lib/dedupePharmacies.js

```diff
@@ -48,6 +48,28 @@ describe('dedupePharmacies', () => {
     });
   });
 
+  describe('when the open item is not one of the 4 nearby items', () => {
+    it('should remove the fourth nearby item', () => {
+      const fourthId = 'AAA04';
+      const services = {
+        open: [{ identifier: 'AAA05' }],
+        nearby: [
+          { identifier: 'AAA01' },
+          { identifier: 'AAA02' },
+          { identifier: 'AAA03' },
+          { identifier: fourthId },
+        ] };
+
+      const result = dedupe(services);
+
+      expect(result.open.length).is.equal(1);
+      expect(result.nearby.length).is.equal(3);
+      result.nearby.forEach((item) => {
```

Comment:
There are collection expectations contains and includes which I'm guessing can be negated. (http://chaijs.com/api/bdd/)